tests/mocks/sqlite_test_db.py

```diff
@@ -1,10 +1,141 @@
+import datetime
 import os
+import uuid
 
 import flask_migrate
 from config import Config
+from db import db
+from db.models.assessment import Assessment
+from db.models.criteria import Criteria
+from db.models.scores_justifications import ScoresJustifications
+from db.models.sub_criteria import SubCriteria
 
 
 class SqliteTestDB:
+
+    crit_1_uuid = uuid.uuid4()
+    crit_2_uuid = uuid.uuid4()
+    crit_3_uuid = uuid.uuid4()
+    sub_crit_1_id = uuid.uuid4()
+    sub_crit_2_id = uuid.uuid4()
+    sub_crit_3_id = uuid.uuid4()
+    sub_crit_4_id = uuid.uuid4()
+    assessment_1_id = uuid.uuid4()
+    assessment_2_id = uuid.uuid4()
```

Comment:
⛏️ can they all be called ..._uuid? for consistency/ clarity wherever they are used?